### PR TITLE
TW-140: Handle photo access permission for image picker

### PR DESCRIPTION
**Pull request by nqhhdev**
- Issue: #140 

- Resolved: 

`IOS`

https://github.com/linagora/twake-on-matrix/assets/99852347/caa2981f-b455-42a5-9858-149e50889f88

`ANDROID`

https://github.com/linagora/twake-on-matrix/assets/99852347/34614008-e63c-4b05-acc2-3143cef50c70

### CODE REVIEW

**Comment by sherlockvn** [comment]
> * Issue: [[Composer] Handle photo access permission for Image picker #140](https://github.com/linagora/twake-on-matrix/issues/140)
> * Resolved:
> 
> `IOS`
> 
>  RPReplay_Final1686677801.MP4 
> `ANDROID` [device-2023-06-14-014756.webm](https://github.com/linagora/twake-on-matrix/assets/99852347/1696dcd7-0182-47d6-8226-bb08ba5e1eae)

IMO, use cmd+shift+5 for demo Android

File: lib/pages/chat/chat.dart [hunk 1]
```diff
@@ -1162,6 +1159,15 @@ class ChatController extends State<Chat> {
         editEvent = null;
       });
 
+  Future<PermissionStatus>? getCurrentPermission() {
```
Comment (hoangdat) [inline lib/pages/chat/chat.dart:1162]:
```suggestion
  Future<PermissionStatus>? getCurrentPhotoPermission() {
```

**Review by hoangdat — COMMENTED** [review]
all is good, complete the final comment and rebase it

**Review by hoangdat — APPROVED** [review]

File: lib/utils/voip/permission_service.dart [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    return _handlePermission(currentStatus);
+  }
+
+  Future<PermissionStatus> _handlePhotosPermissionAndroidAction() async {
+    final currentStatus = await Permission.storage.status;
+    return _handlePermission(currentStatus);
+  }
+
+
+  Future<PermissionStatus> _handlePermission(PermissionStatus currentStatus) async {
```
Comment (sherlockvn) [inline lib/utils/voip/permission_service.dart:35]: `_handlePermission`=> `_handlePhotoPermission` because maybe we have a lot of other permissions

**Review by sherlockvn — APPROVED** [review]

**Pull request merged**